src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -331,6 +332,42 @@ public function move($path, $target)
         return rename($path, $target);
     }
 
+    /**
+     * Create a file from a base64 image
+     *
+     * @param  string  $base64File
+     * @return string
+     */
+    public function fromBase64(string $base64File)
```

Comment:
Is there a reason, this method is not static?

Reply:
Well other functions in the class are not static but I think it is because the File facade handles that

Reply:
Ah, yes. Don't mind me. This question would only make sense if it was a file, not the whole filesystem anyway 🤦🏻‍♂️